specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json

```diff
@@ -822,6 +809,117 @@
         }
       },
       "description": "Response for ListExpressRouteCrossConnection API service call."
+    },
+    "ExpressRouteCrossConnectionPeeringPropertiesFormat": {
```

Comment:
Please review and update attributes that are readOnly in this definition

Reply:
A better name would be `ExpressRouteCrossConnectionPeeringProperties`